### PR TITLE
Real-time digest calculation in `clickhouse-keeper`

### PR DESCRIPTION
**Pull request by antonio2368**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Continuation of #37036 

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

File: src/Coordination/KeeperStorage.cpp [hunk 1]
```diff
@@ -922,18 +1033,11 @@ struct KeeperStorageSetRequestProcessor final : public KeeperStorageRequestProce
...
-                zxid,
-                KeeperStorage::UpdateNodeDelta
-                {
-                    [](KeeperStorage::Node & parent)
-                    {
-                        parent.stat.cversion++;
-                    }
-                }
-        );
+        new_deltas.emplace_back(parentPath(request.path).toString(), zxid, KeeperStorage::UpdateNodeDelta{[](KeeperStorage::Node & parent) {
```
Comment (antonio2368) [inline src/Coordination/KeeperStorage.cpp:1036]: Revert this

File: src/Coordination/KeeperStorage.h [hunk 2]
```diff
@@ -157,8 +190,8 @@ class KeeperStorage
         AuthID auth_id;
     };
 
-    using Operation
-        = std::variant<CreateNodeDelta, RemoveNodeDelta, UpdateNodeDelta, SetACLDelta, AddAuthDelta, ErrorDelta, SubDeltaEnd, FailedMultiDelta>;
+    using Operation = std::
+        variant<CreateNodeDelta, RemoveNodeDelta, UpdateNodeDelta, SetACLDelta, AddAuthDelta, ErrorDelta, SubDeltaEnd, FailedMultiDelta>;
```
Comment (antonio2368) [inline src/Coordination/KeeperStorage.h:194]: brilliant work by `clang-format`

**Comment by antonio2368** [comment]
Failed `Stress test (undefined, actions)` https://s3.amazonaws.com/clickhouse-test-reports/37252/9e5a635da22619bc69f6187d50519733d8b9c02d/stress_test__undefined__actions_.html - cc @kssenii

**Comment by antonio2368** [comment]
@Mergifyio update

**Comment by tavplubix** [comment]
Looks like it breaks `test_keeper_zookeeper_converter`:
https://play.clickhouse.com/play?user=play#c2VsZWN0IAp0b1N0YXJ0T2ZEYXkoY2hlY2tfc3RhcnRfdGltZSkgYXMgZCwKY291bnQoKSwgZ3JvdXBVbmlxQXJyYXkocHVsbF9yZXF1ZXN0X251bWJlciksIGFueShyZXBvcnRfdXJsKQpmcm9tIGNoZWNrcyB3aGVyZSAnMjAyMi0wMy0wMScgPD0gY2hlY2tfc3RhcnRfdGltZSBhbmQgdGVzdF9uYW1lIGxpa2UgJ3Rlc3Rfa2VlcGVyX3pvb2tlZXBlcl9jb252ZXJ0ZXIlJyBhbmQgdGVzdF9zdGF0dXMgaW4gKCdGQUlMJykgZ3JvdXAgYnkgZCBvcmRlciBieSBkIGRlc2M=

**Pull request merged**